src/rws_common.cpp

```diff
@@ -49,6 +49,31 @@ namespace rws
  * Function definitions
  */
 
+std::vector<Poco::XML::Node*> xmlFindNodes(Poco::XML::Node* p_root, const XMLAttribute& attribute)
+{
+  std::vector<Poco::XML::Node*> result;
```

Comment:
```suggestion
  std::vector<Poco::XML::Node*> result{};
```

Reply:
Doesn't initializer lists necessitate C++11?

And what would be the benefit over the other? They both create empty vectors, correct?

Reply:
Yes, that would require C++11.